src/components/books/books.css

```diff
@@ -1,65 +1,355 @@
+:root {
+  /* --medium-blue: #307bbe; */
+  --bubble-gum-pink: #ff5ab7;
+  --black: #1e1e1e;
+  --pinkish-grey: #c7c7c7;
+  --very-light-pink: #c4c4c4;
+  --pale-grey: #f5f6fa;
+  --black-5: rgba(0, 0, 0, 0.05);
+  --z: #fafafa;
+  --black-two: #121212;
+  --white: #e8e8e8;
+  --medium-blue: #307bbe;
+  --cool-blue: #4386bf;
+  --azure: #0290ff;
+  --white-two: #fff;
+}
+
 .parent {
-  background-color: #eaebee;
+  background-color: var(--z);
   padding: 60px 60px;
   display: flex;
   flex-direction: column;
   gap: 20px;
 }
 
+/* Book lists */
+
 .bookli {
-  background-color: #fff;
+  background-color: var(--white-two);
   padding: 20px 20px;
   border: 1px solid #ccc9c9;
   border-radius: 20px;
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
- [ ] Dear @The-Normal-One1, Your design is almost exceptional, But there are some minor details to check. As you can see from the Zeplin design the border for the `bookli` is not visible, instead, it has a very thin box shadow which sounds like a border. This is to make sure you have the exact design clone.
- [ ] Also, You have more `border-radius` value than the Zeplin design. Please consider reducing the value.
- [ ] The other thing is the alignment, If you look closely you will see the left margin for the `bookContainer` is exactly the same as the `BookStore CMS`, and the right is straight to the `User Icon`.
- [ ] Please also, Notice the active navigation item is bolder that the inactive one on the navigation bar. 
> I have marked the differences in red marker. This is to make sure you have the exact clone of the design. 
### Zeplin design 
![image](https://user-images.githubusercontent.com/69077061/222891171-ef6b7ca7-e8cd-4966-a6d1-954227673411.png)
### Your design 
![image](https://user-images.githubusercontent.com/69077061/222891240-ed9bd40a-aa2f-445e-8f1e-ab9e1a36dcca.png)